installer/roles/local_docker/templates/nginx.conf.j2

```diff
@@ -47,7 +47,12 @@ http {
     {%endif %}
 
     server {
-        {% if ssl_certificate is defined %}
+        {% if (ssl_certificate is defined) and (ssl_certificate_key is defined) %}
+        listen 8053 ssl;
+
+        ssl_certificate /etc/nginx/awxweb.pem;
+        ssl_certificate_key /etc/nginx/awxweb_key.pem;
+        {% elif if (ssl_certificate is defined) and (ssl_certificate_key is not defined) %}
```

Comment:
Is this a syntax error? `elif if` seems wrong.

Reply:
hm. didn't noticed it. Checking now